navit/maptool/CMakeLists.txt

```diff
@@ -1,16 +1,44 @@
+find_package(Protobuf-c REQUIRED)
```

Comment:
This should be in the if statement, so that navit without maptool can build even without protobuf